file-format-conversions.md

```diff
@@ -0,0 +1,160 @@
+* This should, in principle, also be possible for PDF-A 2b and 3b, but we have not determined how to do this.
+
+
+
+### {`.doc`, `.docx`} → `.txt`
+
+```
+$ soffice --convert-to txt:Text INPUT_FILE
+```
+
+#### Notes
+
+* This does not work for `.pdf` files. The command `soffice --infilter="writer_pdf_import" --convert-to txt:Text INTPUT_FILE` runs but creates an empty plaintext file, even for very simple PDFs. The pipeline `.pdf` → `.doc` → `.txt` has the same behavior.
+
+
+
+### {`.xls`, `.xlsx`} → `.tsv`
+
+```
+$ soffice --convert-to tsv:"Text - txt - csv (StarCalc)":9,34,0 INPUT_FILE
+```
+
+#### Notes
+
+* See [this page](https://wiki.openoffice.org/wiki/Documentation/DevGuide/Spreadsheets/Filter_Options#Filter_Options_for_the_CSV_Filter) for details on the filter options (`9` here is the ASCII value for horizontal tab).
+
+
+
+### {`.gif`, `.bmp`, `.jpg`} → `.tif`
+
+```
+$ soffice --convert-to tif INPUT_FILE
+```
+
+
+
+## PDF2ARCHIVE
+
+### `.pdf` → PDF-A 1b
+
+```
+ $ ./pdf2archive --validate INPUT_FILE OUTPUT_FILE_NAME
+```
+
+#### Notes
+
+* All directories in the path `OUTPUT_FILE_NAME` must exist; `pdf2Archive` does not create directories.
+* The input file is expected to be in the same place as the script; in the least, the script seems to want its own local copy of `verapdf` for validation. It may be very simple to fork the project and fix this.
+* There are files which LibreOffice failed to make compliant that `pdf2Archive` handled, and vice versa. **It may be worth using both and checking if either succeeds against `verapdf`.**
+
+
+
+## Pandoc
+
+### `.docx` → {`.pdf`, `.txt`}
+
+```
+$ pandoc INPUT_FILE -o OUTPUT_FILE_NAME
+```
+
+#### Notes
+
+* Pandoc determines the which conversion to used based on the `INPUT_FILE` and the extension of `OUTPUT_FILE_NAME`.
+* These seem to be the only relevant conversions for Pandoc ([This page](https://pandoc.org/MANUAL.html#options) has all possible input/output file types). With this, we have the pipeline `.docx` → `.pdf` → PDF-A 1b.
+* All directories in the path `OUTPUT_FILE_NAME` must exist; Pandoc does not create directories.
+
+
+
+## ImageMagick
+
+### {`.gif`, `.bmp`, `.jpg`} → `.tif`
+```
+$ convert INPUT_FILE OUTPUT_FILE_NAME
+```
+
+#### Notes
+
+* Like Pandoc, ImageMagick performs implicit file conversion.
+* All directories in the path `OUTPUT_FILE_NAME` must exist; `convert` does not create directories.
+
+
+
+## libvips
+
+### {`.gif`, `.bmp`, `.jpg`} → `.tif`
+
+```
+$ vips copy INPUT_FILE OUTPUT_FILE_NAME
+```
+
+#### Notes
+
+* Like Pandoc, libvips performs implicit file conversion.
+* All directories in the path `OUTPUT_FILE_NAME` must exist; libvips does not create directories.
+
+
+
+## pdftotext
+
+### `.pdf` → `.txt`
+
+```
+$ pdftotext INPUT_FILE OUTPUT_FILE_NAME
+```
+
+#### Notes
+
+* All directories in the path `OUTPUT_FILE_NAME` must exist; libvips does not create directories.
```

Comment:
I think this bullet point was intended for the `lipvips` section?

Reply:
Absolutely. Just pushed a fix to that. Thanks for catching it.